vault/changelog.release.2022-06-07.md

```diff
@@ -0,0 +1,100 @@
+
+- 💡`kevins8#0590` shared information about showing the difference between multiple commits on github using github.com/<username>/<repo_name>/compare/<commit1>...<commit2>. [See Example](https://github.com/dendronhq/handbook/compare/6adc8f73e8abe060541d209daf5005dd730eb456..532f429bfdd9a293f7c1bbca3805c16ef84ea48f). 
+
+
+### Dendron Reading Series
+
+This week's entry in the [[Dendron Reading Series|dendron://dendron.dendron-site/community.events.reading-series]].
+
+[[What happened to tagging?|dendron://dendron.dendron-site/community.events.reading-series.2022.06.07]]
+
+### Event Reminders
+
+- **Greenhouse Talks:** Visit the [[Greenhouse Talks|dendron://dendron.dendron-site/community.events.greenhouse]] for notes from previous sessions.
+    - Subject: _Getting Things Done (GTD) and Other Task-Management Workflows (with Dendron!)_
+    - Description: _Interested speakers from the community will take 5 - 10 minutes each to present their workflows for managing actionable information, followed by an open discussion. The event will be recorded and later published online._
+    - Next: [Fri, Jun 24, 04:00 PM PST / 00:00 UTC](https://link.dendron.so/luma)
+    - [Greenhouse Talk Recordings - YouTube Playlist](https://link.dendron.so/greenhouse)
+- **Community Teatime:** Drink tea, eat food, and hangout with the Dendron crew!
+    - Next: [Thu, Jun 09, 11:00 AM PST / 19:00 UTC](https://link.dendron.so/luma)
+- **New User Tuesdays:** Visit the [[New User Tuesdays page|dendron://dendron.dendron-site/community.events.new-user-tuesdays]] for notes from previous sessions.
+    - Next: [Tue, Jun 14, 8:30 AM PST / 16:30 UTC](https://link.dendron.so/luma)
+
+### Thank You's
+
+A big **thanks** to the following gardeners that brought up issues, contributions, and fixes to this release :man_farmer: :woman_farmer: 
+Visit [[Discord Roles|dendron://dendron.dendron-site/community.discord.roles]] for more information.
+
+- [lordfrikk]() `@lordfrikk#3247`
+    - #role.taxonomist
+    - [wrong example in `nav_exclude_children`](https://wiki.dendron.so/notes/4l0mnzhwcos8kdn5aif1we9/)
+
+- [Marek Lukasiewicz](https://github.com/Maarrk) `@Maarrk#9537`
+    - #role.horticulturalist
+    - [enhance: Add axios to hook function](https://github.com/dendronhq/dendron/pull/3001)
+
+- [Riccardo Dozzo](https://github.com/rioka) `@rioka#2156`
+  - #role.bugcatcher
+  - [OS dependent path for vaults](https://github.com/dendronhq/dendron/issues/3040)
+
+- [bugfixd](https://github.com/bugfixd)
+  - #role.bugcatcher
+  - [Unable to prevent Dendron from treating @ as a user tag by escaping it](https://github.com/dendronhq/dendron/issues/3039)
+
+- [Tika](https://github.com/SR--) `Tika#9526`
+  - [Incorrect handling of colons after the task square brackets](https://github.com/dendronhq/dendron/issues/3037)
+  
+- [Ryan Hill](https://github.com/rlh1994) `@rlh1994#9754`
```

Comment:
can we add the following?

- [Mihai](https://github.com/mihaiconstantin) `mihaiconstantin#9440`
  - #role.speaker
  - speaking at the last Dendron greenhouse!